kas/machine/qemuarm64.yml

```diff
@@ -18,5 +18,6 @@ local_conf_header:
   machine-avocado-qemuarm64: |
     ## Debugging prior to extensions loading
     #AVOCADO_ALLOW_ROOT_LOGIN = "1"
+    ROOTFS_IMAGE_EXTRA_INSTALL:append = " systemd-zram-generator"
```

Comment:
Let's go about this a different way. Instead of adding this to each individual config, lets add this as a defaut rootfs / initramfs images. We can make it depend on a DISTRO_FEATURE of `zram`. For systems we don't want to enable zram, that machine conf can remove from DISTRO_FEATURES.